### ISSUE TITLE
 the sign is put but I can't buy

### ISSUE THREAD

**Issue by Claralaqueen** [NLthijs48/AreaShop#553]
Hello I did download the version you sent to problems 552 it seems to me (excuse me in advance I am French I try to be understandable) so I resume so I no longer have the message bug when I done / as addsign the sign create well but when I want to buy or rent it does not matter I clicked nothing happens nothing helped me please my server is in 1.14.4
![Capture](https://user-images.githubusercontent.com/64494895/80524007-3e7e1380-898f-11ea-8426-a6fe56360984.JPG)

**Comment by Syrkulous** [comment]
You may need to find an older build or reset the sign

**Comment by Claralaqueen** [comment]
Old version does not work its the error message bug when I order /as addsign

**Comment by Syrkulous** [comment]
Use the version I sent originally
Make sure to restart the server and remove any older .jars of the plugin if you haven't

**Comment by Claralaqueen** [comment]
Can return this version I do not speak which you will speak

**Comment by Claralaqueen** [comment]
Can return this version I don't see which one you will be talking about

**Comment by Claralaqueen** [comment]
Can send me this version I don't see which one you will talk about

**Comment by Syrkulous** [comment]
http://wiefferink.me:8080/job/AreaShop/299/

**Comment by Claralaqueen** [comment]
I'll try it right away I'll tell you in a few minutes

**Comment by Claralaqueen** [comment]
it still does not work when i click on it does nothing however the command / as addsign displays the sign well but I can not interact with :(

**Comment by Syrkulous** [comment]
It's possible you removed a permission from yourself or others that allows you to interact with the sign, you haven't reset the sign, or you are using FAWE but if the plugin is loading, I don't believe the third issue is active

**Comment by Claralaqueen** [comment]
how to give this authorization I redid all the region and the panels and I had to delete all the areashop file and gave it back after restarting the server here is my list of plugins
![2020-04-29_11 15 11](https://user-images.githubusercontent.com/64494895/80580026-31066f00-8a0b-11ea-92d3-3da0c054f1ee.png)
![2020-04-29_11 15 23](https://user-images.githubusercontent.com/64494895/80580031-32d03280-8a0b-11ea-8e0e-2f72e5b7d772.png)
![2020-04-29_11 19 31](https://user-images.githubusercontent.com/64494895/80580143-60b57700-8a0b-11ea-8d3a-9a2a762506ae.png)

**Comment by Syrkulous** [comment]
If you've already removed the regions you had.
Try removing the plugin and it's files entirely,
Then restart the server.
Then add the AreaShop plugin back and restart again
Make sure you're using the  version I sent.

If this does not work,
Make sure you have the right permissions set in your permissions plugin, and make sure world guard and world edit are updated

**Comment by Claralaqueen** [comment]
I don't understand why it doesn't work it annoys me

**Comment by Syrkulous** [comment]
I don't understand either, first time dealing with this issue

**Comment by Claralaqueen** [comment]
it is little to be the permission to interact with the sign I don't know how to add it, can you explain it to me?

**Comment by Syrkulous** [comment]
From the plugins list you gave me, it doesn't seem like you have a permissuing plugin. So that might cut out that issue. But I could be wrong.  For a permission plugon, I use luckperms though. It's the easiest for me to use and is easy to work with.

Are you 100% you've followed everything I've said? When you right click the sign it doesn't mark rented at all? And you've attempted to use older versions of the plugin and they still didn't work?

**Comment by Claralaqueen** [comment]
yes i tried to use the older versions still the same i think it is a permission problem i will install your plugins and put the peromissions of sign i tell you its after

**Comment by Claralaqueen** [comment]
actually it was very simple I just didn't have the right economy plugin don't take gringotts
